player/lua/stats.lua

```diff
@@ -945,7 +945,7 @@ local function add_audio(s)
     local merge = function(r, ro, prop)
         local a = r[prop] or ro[prop]
         local b = ro[prop] or r[prop]
-        return (a == b or a == nil) and a or (a .. " → " .. b)
+        return (a == b or a == nil) and a or (a .. " \xE2\x9E\x9C " .. b)
```

Comment:
`\x` escapes in Lua are a LuaJIT extension IIRC

Reply:
Ok, right for lua we should use chars directly.